pkg/k8shandler/deployment.go

```diff
@@ -40,7 +38,7 @@ type deploymentNode struct {
 	esClient elasticsearch.Client
 }
 
-func (deploymentNode *deploymentNode) populateReference(nodeName string, node api.ElasticsearchNode, cluster *api.Elasticsearch, roleMap map[api.ElasticsearchNodeRole]bool, replicas int32, client client.Client, esClient elasticsearch.Client) {
+func (node *deploymentNode) populateReference(nodeName string, n api.ElasticsearchNode, cluster *api.Elasticsearch, roleMap map[api.ElasticsearchNodeRole]bool, replicas int32, client client.Client, esClient elasticsearch.Client) {
```

Comment:
`node` or follow the pattern of our other objects and use `dn` ?

Reply:
or was this just to appease lint? if so we can address my comment in https://issues.redhat.com/browse/LOG-824

Reply:
They are all `node` now. I can push a separate commit to rename them to hopefully reduce noise in the review.